src/wavm.cpp

```diff
@@ -194,6 +194,35 @@ namespace wavm_host_module {
   };
 } // namespace wavm_host_module
 
+
+void WavmEngine::verifyContract(vector<uint8_t> const& code)
```

Comment:
@chfast I'm not sure how to do this without code duplication given the mess `GCPointer` is causing. Any ideas?

Reply:
What exactly is duplicated?

Reply:
This entire body below is present in `internalExecute`.